src/Arcus.Templates.ServiceBus.Topic/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0.101-alpine3.9 AS base
+WORKDIR /app
+EXPOSE 80
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0.101-alpine3.9 AS build
```

Let's include all security patches
```suggestion
FROM mcr.microsoft.com/dotnet/core/sdk:3.0-alpine3.10 AS build
```